components/tabs/index.tsx

```diff
@@ -153,6 +160,7 @@ export default class Tabs extends React.Component<TabsProps, any> {
 
     const renderTabBar = () => (
       <ScrollableInkTabBar
+        inkBarAnimated={inkBarAnimated}
```

Comment:
I cannot find documentation about `ScrollableInkTabBar[inkBarAnimated]`, could you show me the link?